waargonaut.cabal

```diff
@@ -10,7 +10,7 @@ name:                waargonaut
 -- PVP summary:      +-+------- breaking API changes
 --                   | | +----- non-breaking API additions
 --                   | | | +--- code changes with no API change
-version:             0.7.0.1
+version:             0.7.1.0
```

Comment:
Please make the version `0.8.0.0`. Constructors were added to exported sum types, which is technically a breaking change.